docs/index.md

```diff
@@ -13,14 +13,39 @@ We built PydanticAI with one simple aim: to bring that FastAPI feeling to GenAI
 
 ## Why use PydanticAI
 
-* Built by the team behind Pydantic (the validation layer of the OpenAI SDK, the Anthropic SDK, LangChain, LlamaIndex, AutoGPT, Transformers, CrewAI, Instructor and many more)
-* [Model-agnostic](models.md) — currently OpenAI, Anthropic, Gemini, Ollama, Groq, and Mistral are supported, and there is a simple interface to implement support for other models.
-* [Type-safe](agents.md#static-type-checking)
-* Control flow and agent composition is done with vanilla Python, allowing you to make use of the same Python development best practices you'd use in any other (non-AI) project
-* [Structured response](results.md#structured-result-validation) validation with Pydantic
-* [Streamed responses](results.md#streamed-results), including validation of streamed _structured_ responses with Pydantic
-* Novel, type-safe [dependency injection system](dependencies.md), useful for testing and eval-driven iterative development
-* [Logfire integration](logfire.md) for debugging and monitoring the performance and general behavior of your LLM-powered application
+:material-account-group:{ .lg .middle .team-blue }&nbsp;&nbsp;<strong class="vertical-middle">Built by the Pydantic Team</strong>
+
+Built by the team behind [Pydantic](https://docs.pydantic.dev/latest/) (the validation layer of the OpenAI SDK, the Anthropic SDK, LangChain, LlamaIndex, AutoGPT, Transformers, CrewAI, Instructor and many more).
+
+:fontawesome-solid-shapes:{ .lg .middle .shapes-orange }&nbsp;&nbsp;<strong class="vertical-middle">Model-agnostic</strong>
+
+Supports OpenAI, Anthropic, Gemini, Ollama, Groq, and Mistral, and there is a simple interface to implement support for [other models](models.md).
+
+:logfire-logo:{ .lg .middle }&nbsp;&nbsp;<strong class="vertical-middle">Pydantic Logfire Integration</strong>
```

Comment:
I'd make these smaller, `.md` or even `.sm` - maybe try both and see what you think. You can hide a lot of design flaws by reducing the screen real estate things get 😆 I think the icons we get with mkdocs aren't great, and it's ok to use them here but we don't want them to draw too much focus or for the user to notice the inconsistencies in colors too much...

Reply:
Good call, I like medium!